library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -135,7 +135,8 @@
     }
 
     select.form-control {
-        -webkit-appearance: menulist;
+        -webkit-appearance: none;
```

Comment:
Use "@include appearance(none);" instead of "-webkit-appearance: none;"

Reply:
is `@include` needed or would `appearance:none;` work?

Reply:
Hi @profmikegreene , to be honest I'm not completly sure if it is necessary, this is to make all browser compatible with this css property. If we use this sakai mixin for this, I'm sure that it will work properly.

Reply:
Using that `@include` tag it will use these two css properties https://github.com/sakaiproject/sakai/blob/ec8ee1cef6e2d7dffb5908baa492c504d8d65f4e/library/src/morpheus-master/sass/base/_compass.scss#L36